### ISSUE TITLE
[Accessibility]: Missing last character in some words in screen reader announcements on upload information page

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#44556]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

The text announcements for some bits of text on the upload information page are incorrect, the last character is omitted. Specifically, "commit" becomes "commi" and "by" becomes "b". Not sure why this is happening.

### Expected behavior

The text should be read out correctly.

### Additional details

https://user-images.githubusercontent.com/93103176/202493776-8fee4bbd-21eb-4c70-b24d-63526fde5d28.mov

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
I can't repro this in Chrome, I think this may be a Firefox bug. I don't think there is something we can do.
